dts/common/nordic/nrf54h20.dtsi

```diff
@@ -333,10 +333,12 @@
 			ranges = <0x0 0x5f000000 0x1000000>;
 
 			usbhs: usbhs@86000 {
-				compatible = "snps,dwc2";
+				compatible = "nordic,nrf-usbhs", "snps,dwc2";
 				reg = <0x86000 0x1000>, <0x2f700000 0x40000>;
 				reg-names = "wrapper", "core";
 				interrupts = <134 NRF_DEFAULT_IRQ_PRIORITY>;
+				num-in-eps = <12>;
```

Comment:
This is not exactly right. While nRF54H20 has the possibility to use 12 IN endpoints, it can only support EP0 plus any combination of up to 7 different IN endpoints (see `INEps` in ` GHWCFG4`). The stack expects that all endpoints can be used simultaneously, which is not the case here.

I would go with 8 here, possibly with a comment that it is limited due to number available of TxFIFOs.